packages/graph/hash_graph/lib/graph/src/api/rest/middleware.rs

```diff
@@ -196,7 +215,36 @@ pub fn span_maker(request: &hyper::Request<hyper::Body>) -> tracing::Span {
         http.client_ip = %client_ip,
         http.user_agent = %user_agent,
         http.host = %host,
+        http.status_code = Empty,
+        otel.status_code = Empty,
     );
     tracing_opentelemetry::OpenTelemetrySpanExt::set_parent(&span, remote_context);
     span
 }
+
+fn span_on_response(
+    response: &hyper::Response<axum::body::BoxBody>,
```

Comment:
I believe this is redundant too
```suggestion
    response: &Response<axum::body::BoxBody>,
```